qubes/tests/storage.py

```diff
@@ -24,6 +24,7 @@
 from qubes.storage import pool_drivers
 from qubes.storage.file import FilePool
 from qubes.storage.reflink import ReflinkPool
+from qubes.storage.zfs import ZFSQpool
```

Comment:
It's nice you import it here, but some actual tests would be even nicer ;)

Reply:
Ideally, there would be general tests for all storage pools.  @marmarek thoughts?